### DISCUSSION TITLE
Build Failure due to failed tests

### DISCUSSION THREAD

**Discussion by Kaustubh251002** [checkstyle/checkstyle#11225] [Q&A] [answered]
I wanted to try contributing to checkstyle since I am familiar with Java and Git, and followed the instructions to download and unzip it, but while running "mvn install" in the root folder I am getting a build failure due to 5 failed tests out of 3701.
Following are the errors generated. I would highly appreciate some guidance on how to start contributing to this project.
![image](https://user-images.githubusercontent.com/72098934/150351071-8a4da344-d402-421a-be72-b2aa1e8e8e72.png)
![image](https://user-images.githubusercontent.com/72098934/150351166-b3f07509-9d65-4f4b-95d3-23004e9737f8.png)
![image](https://user-images.githubusercontent.com/72098934/150351239-5457e1fd-2a29-4da2-9b01-4156840db410.png)
![image](https://user-images.githubusercontent.com/72098934/150351303-4597c9f2-e110-4d28-8f05-2122c14fa019.png)
![image](https://user-images.githubusercontent.com/72098934/150351368-707fbab5-bd24-40a3-8e1a-fbe3cda15f15.png)
![image](https://user-images.githubusercontent.com/72098934/150351413-b619dccf-41b9-471f-99ee-f9d5ea236bb1.png)
![image](https://user-images.githubusercontent.com/72098934/150351445-98264e0a-aeec-49f7-8af4-b2e04a9ad29e.png)
![image](https://user-images.githubusercontent.com/72098934/150351494-88b5721f-08b5-4eb9-af30-fd0bbe0f1c33.png)

**Comment by romani** [comment, marked as answer]
Looks like line ending problem
Make sure you clone repo with such git settings
https://github.com/checkstyle/checkstyle/blob/f5419e19e7a443ef31de2288aee9740b3f2de1d9/appveyor.yml#L23

**Reply by Kaustubh251002** [reply]
Thanks a lot!

**Reply by rnveach** [reply]
@romani I thought this requirement was removed in https://github.com/checkstyle/checkstyle/issues/6558 . We specifically removed documentation for this in that PR, see https://github.com/Vampire/checkstyle/commit/5bc05a964a6161b881745c69e6df75b3858e97ed .
I assumed this was why there was no way to reproduce https://github.com/checkstyle/checkstyle/issues/8884 .

It doesn't really make sense the answer to a question like this is to use `core.autocrlf` when we said we removed the requirement for it.

**Reply by romani** [reply]
We need window user to report issue in this case and we need window user to provide insight on how to handle it
